python/ray/serve/api.py

```diff
@@ -1414,49 +1425,23 @@ def list_deployments() -> Dict[str, Deployment]:
     return deployments
 
 
+@PublicAPI
```

Comment:
let's not make this a public API (this means it's documented and stable). for now this is only used in CLI.